website/docs/r/region.html.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_region"
+sidebar_current: "docs-vcfa-resource-region"
+description: |-
+  Provides a resource to manage Regions in VMware Cloud Foundation Automation.
+---
+
+# vcfa\_region
+
+Provides a resource to manage Regions in VMware Cloud Foundation Automation.
+
+~> Only `System Administrator` can create this resource.
+
+## Example Usage
+
+```hcl
+data "vcfa_vcenter" "one" {
+  name = "vcenter-one"
+}
+
+data "vcfa_supervisor" "one" {
+  name       = "my-supervisor-name"
+  vcenter_id = data.vcfa_vcenter.one.id
+}
+
+data "vcfa_nsx_manager" "main" {
+  name = "nsx-manager-one"
+}
+
+resource "vcfa_region" "one" {
+  name                 = "region-one"
+  nsx_manager_id       = data.vcfa_nsx_manager.main.id
+  supervisor_ids       = [data.vcfa_supervisor.one.id]
+  storage_policy_names = ["vSAN Default Storage Policy"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A name for Region. It must match RFC 1123 Label name (lower-case alphabet,
+  numbers between 0 and 9 and hyphen `-`)
```

Comment:
Added hyphen at the end, but got validation error:

![Screenshot 2025-01-22 at 11 52 52](https://github.com/user-attachments/assets/fa4450dd-229a-4a36-801c-bca9a3b495b8)

Reply:
> RFC 1123 Label Names. Some resource types require their names to follow the DNS label standard as defined in RFC 1123.

It makes sense, you can't end a dns name with `-` as well